<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartUp</class>
 <widget class="QDialog" name="StartUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>SELECT STACK SIZES FOR DIFFERENT MODES</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="leFiqSize"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>IRQ_STACK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leIrqSize"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leUndSize"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>USR_STACK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>UND_STACK</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leSvcSize"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ABRT_STACK</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>SVC_STACK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>FIQ_STACK</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leAbrtSize"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="leUsrSize"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="cbIRQFunctions">
     <property name="text">
      <string>Add EnableIRQ and disableIRQ Functions</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pbPreview">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbHelp">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>pbPreview</zorder>
  <zorder>pbApply</zorder>
  <zorder>pbCancel</zorder>
  <zorder>groupBox</zorder>
  <zorder>pbHelp</zorder>
  <zorder>cbIRQFunctions</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
